tests/suite/utils/custom_resources_utils.py

```diff
@@ -125,7 +125,7 @@ def read_ts(custom_objects: CustomObjectsApi, namespace, name) -> object:
     """
     Read TransportService resource.
     """
-    return read_custom_resource_v1alpha1(custom_objects, namespace, "transportservers", name)
+    return read_custom_resource_v1(custom_objects, namespace, "transportservers", name)
```

Comment:
think we can now replace the usage of this func with https://github.com/nginxinc/kubernetes-ingress/blob/main/tests/suite/utils/custom_resources_utils.py#L63, basically all tests that used `read_custom_resource_v1alpha1` can use the one in link above

Reply:
That's a good point. I'll change the name of this one back and do the usage update.